Objects/moduleobject.c

```diff
@@ -261,6 +264,8 @@ PyModule_FromDefAndSpec2(PyModuleDef* def, PyObject *spec, int module_api_versio
     PyObject *m = NULL;
     int has_multiple_interpreters_slot = 0;
     void *multiple_interpreters = (void *)0;
+    int has_gil_slot = 0;
+    void *gil_slot = Py_MOD_GIL_USED;
```

Comment:
Please be sure to address the compiler warnings here.

```suggestion
#ifdef Py_GIL_DISABLED
    void *gil_slot = Py_MOD_GIL_USED;
#endif
```

Reply:
I'll add a cast to void instead. I had one in a previous version of the PR but it got lost in some shuffling.